faq.html

```diff
@@ -221,13 +231,14 @@ <h4 class="panel-title">
           <div id="collapse6" class="panel-collapse collapse">
             <div class="panel-body">
               <p>
-                Our required annual budget is usually between $50,000 and $75,000, depending on team size and if we qualify for
-                the <a href="/glossary#champs" target="_blank">championship</a> tournament in Detroit (we have the past 8 years).
-                Most of this budget is funded by our generous sponsors, and Irondale High School pays for two of our busses to
-                competitions. The rest is paid by our students' activity registration fee, which is <strong>$50-$100 per person</strong>.
-                At the end of October, when students officially sign up to join the team, they can choose to pay the fee all at once or throughout the
-                course of the year. Out-of-town competitions are not included in this fee. Those cost an <strong>additional $150-$200 each</strong>.
-                Fees are waived for any student with free or reduced lunch.
+                Our required annual budget is usually between $80,000 and $90,000, depending on team size and if we qualify for
+                the <a href="/glossary#champs" target="_blank">championship</a> tournament in Houston (we have been since 2013 years).
```

Comment:
```suggestion
                the <a href="/glossary#champs" target="_blank">championship</a> tournament in Houston (we have qualified every year since 2013).
```